h5f.go

```diff
@@ -66,7 +56,8 @@ func CreateFile(name string, flags int) (*File, error) {
 	return newFile(hid), nil
 }
 
-// Opens an existing HDF5 file.
+// Open opens and returns an an existing HDF5 file. The returned
```

Comment:
Just a general thought: should we maybe stay consistent with the method/function comments and start them with method/function name. Right now it seems to be the case for half of the ones below.

Reply:
Yeah, I missed some. The intention was to do "Func does ....". We can fix more later. There's other lint I want to remove elsewhere, so I'm fine with leaving this and repolishing.

Reply:
SGTM